pkg/postage/postagecontract/contract.go

```diff
@@ -66,6 +66,7 @@ func New(
 	transactionService transaction.Service,
 	postageService postage.Service,
 	postageStorer postage.Storer,
+
```

Comment:
Please remember to do a self-review once you post your diff. All these changes not pertaining to your changeset need to be reverted once you are done.